bot-components/GitHub_app.ml

```diff
@@ -89,3 +89,15 @@ let get_installation_token ~bot_info ~key ~app_id ~owner ~repo =
       get_installation_token ~bot_info ~jwt ~owner ~repo
   | Error e ->
       Lwt.return (Error e)
+
+let get_installations ~bot_info ~key ~app_id =
+  match make_jwt ~key ~app_id with
+  | Ok jwt ->
+      generic_get ~bot_info:{bot_info with github_token= JWT jwt}
```

Comment:
Why use the `generic_get` function here (at the cost of having to add the new `JWT` constructor to the `github_token` type)? Doesn't the `get` function that it is defined in this file work?